app/router/transfer-reset/transfer-reset.go

```diff
@@ -0,0 +1,58 @@
+package transfer_reset
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/go-chi/chi"
+	"github.com/go-chi/render"
+	"github.com/limechain/hedera-eth-bridge-validator/app/domain/service"
+	"github.com/limechain/hedera-eth-bridge-validator/app/helper/metrics"
+	transferModel "github.com/limechain/hedera-eth-bridge-validator/app/model/transfer"
+	"github.com/limechain/hedera-eth-bridge-validator/app/router/response"
+	"github.com/limechain/hedera-eth-bridge-validator/config"
+)
+
+var (
+	Route  = "/transfer-reset"
+	logger = config.GetLoggerFor(fmt.Sprintf("Router [%s]", Route))
+)
+
+func NewRouter(transferService service.Transfers, prometheusService service.Prometheus, nodeConfig config.Node) chi.Router {
+	r := chi.NewRouter()
+	r.Post("/", transferReset(transferService, prometheusService, nodeConfig))
+	return r
+}
+
+// POST: .../transfer-reset
+func transferReset(transferService service.Transfers, prometheusService service.Prometheus, nodeConfig config.Node) func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Can we change this so it takes only the access token, not the whole node config.